package.json

```diff
@@ -47,6 +47,7 @@
     "redux-saga": "^1.1.3",
     "redux-smart-actions": "^1.0.0",
     "reselect": "^4.0.0",
+    "styled-components": "^5.3.0",
```

Comment:
I think that it is not worth adding these packages, since our project is already using the approach with the material UI